clang/lib/Sema/AnalysisBasedWarnings.cpp

```diff
@@ -983,10 +983,9 @@ static void DiagUninitUse(Sema &S, const VarDecl *VD, const UninitUse &Use,
   case UninitUse::AfterDecl:
   case UninitUse::AfterCall:
     S.Diag(VD->getLocation(), diag::warn_sometimes_uninit_var)
-      << VD->getDeclName() << IsCapturedByBlock
-      << (Use.getKind() == UninitUse::AfterDecl ? 4 : 5)
-      << const_cast<DeclContext*>(VD->getLexicalDeclContext())
```

Comment:
IT looks like this one goes back to 2013, so I am guessing it used to be const and that changed and this was just missed. The git archeology is not that hard it just requires a few `--ignore-rev`'s. It would of course be nice if this actually worked on github instead of timing out.